packages/solar-simulator/package.json

```diff
@@ -19,6 +19,7 @@
   "license": "GPL-3.0-only",
   "dependencies": {
     "@energyweb/asset-registry": "1.3.0",
+    "@energyweb/market": "1.3.0",
```

Comment:
When adding dependency here please also add it to tsconfig as dependent project to keep the consistency.